app-backend/batch/src/main/scala/com/azavea/rf/batch/export/spark/Export.scala

```diff
@@ -50,7 +50,7 @@ object Export extends SparkJob with Config with LazyLogging {
     projLocs: Map[UUID, List[(UUID, String)]],
     conf: HadoopConfiguration
   )(implicit sc: SparkContext): Unit = {
-    interpretRDD(ast, params.sources, ed.input.resolution, sceneLocs, projLocs) match {
+    interpretRDD(ast, params.sources, params.overrides, ed.input.resolution, sceneLocs, projLocs) match {
```

Comment:
Good point - we can handle resolution of fallbacks prior to attempting to actually use `EvalParams`